framer/LayerStateMachine.coffee

```diff
@@ -21,7 +21,7 @@ class exports.LayerStateMachine extends BaseClass
 		get: -> @_currentName
 
 	@define "previousName",
-		get: -> _.last(@_previousNames)
+		get: -> _.last(@_previousNames) or "default"
```

Comment:
I think it's better to _always_ return something here so you don't have to do a null check (if you just created a layer and ask for `layer.states.previous.x`.